README.md

```diff
@@ -40,21 +40,22 @@
   - [`goal_src/`](#goal_src)
   - [`game` runtime](#game-runtime)
 
-## READ FIRST
+## Please read first
 
-The github repositories are for development of the project and tracking active issues.  Most of the information you will find here pertains to setting up the project for development purposes and is not relevant to a general user.
+Our repositories on GitHub are primarily for development of the project and tracking active issues. Most of the information you will find here pertains to setting up the project for development purposes and is not relevant to the end-user. The Releases page will provide you with a pre-compiled set of tools you need to get this project running on your machine. 
 
-There is a discord server where discussion occur https://discord.gg/VZbXMHXzWv
+For questions or additional information pertaining to the project, we have a Discord for discussion here: https://discord.gg/VZbXMHXzWv
 
-Additionally you can find further information and answers to **frequently asked questions** on the project's main website https://open-goal.github.io/
+Additionally, you can find further documentation and answers to **frequently asked questions** on the project's main website: https://open-goal.github.io/
 
-### How to play the game
+**Do not use this decompilation project without providing your own legally purchased copy of the game.**
 
-To just play the game, follow the steps in this video https://www.youtube.com/watch?v=p8I9NfuZOgE
+### Quick Start
+If you just want to play the game, you can follow the steps in this video: https://www.youtube.com/watch?v=yQBKCnS5MDY
 
-We don't save any assets from the game - you must use your own legitimately obtained copy of the game.
+We do not use any assets from the game - you must use your own legitimately obtained PS2 copy of the game. We support every retail PAL, NTSC, and NTSC-J build, including Greatest Hits copies. 
 
-> Note that this installation method is a temporary stop gap while we work on something more user-friendly.
+> Please be aware that this installation method is a temporary stop gap while we work on something more user-friendly.
```

Comment:
now that we have something more user friendly (for windows atleast) i think this can be completely removed.